### ISSUE TITLE
problems without \nextproblem leads to missing number with xelatex

### ISSUE THREAD

**Issue by thorstengrothe** [Strauman/exercisebank#82]
Take `firstexercise` of the examples in the last prerelease package. These exercises do not have
the `\nextproblem` tag before `\begin{problem}`. The compilation leads to this error

`Missing number, treated as zero. \begin{problem}`

but only with `xelatex`, with `pdflatex` it compiles without an error. If you set `\nextproblem{points=6}` it works with `xelatex` too. If you set `\nextproblem{tag=a}` the error comes again.

Is this a bug?

Thorsten

**Comment by Strauman** [comment]
Interesting. It's definitely not supposed to be like this? `xelatex`

**Comment by Strauman** [comment]
I found the error. And it's huge. `\pdfstrcmp` does not work in `xelatex`. I use it everywhere. Darn.

**Comment by thorstengrothe** [comment]
Hm, sounds not so good but I'm glad that I was right. Maybe this could help, I found a discussion about `\pdfstrcmp` between some contributors well known in the latex scene.

[Link](https://tex.stackexchange.com/questions/111296/pdfstrcmp-or-strcmp-in-pure-texurl)

Thorsten

**Comment by Strauman** [comment]
Yup. I saw that as well. I guess I have to find some other solution at some point.
